File: iwannabetheguy.java
import java.util.Scanner;
import java.util.Set;
import java.util.HashSet;

public class iwannabetheguy{

	static Scanner in = new Scanner(System.in);

	public static void main(String[]args){
	
		Set <Integer> nums = new HashSet <Integer>();
		int n = in.nextInt();
		int p = in.nextInt();
		
		for(int i = 0; i < p; i++){
			
			nums.add(in.nextInt());

		}
		
		int q = in.nextInt();
		
		for(int i = 0; i < q; i++){
			
			nums.add(in.nextInt());
			
		}

		if(nums.size() == n){
		
			System.out.println("I become the guy.");
		}

		else{
		
			System.out.println("Oh, my keyboard!");
		
		}
		}
	}

File: iwannabetheguy.py
n = int(input())
x = list(map(int, input().split()))
y = list(map(int, input().split()))

x.pop(0)
y.pop(0)

myset = set(x + y)

if n == len(myset):
    print("I become the guy.")

else:
    print("Oh, my keyboard!")
